### DISCUSSION TITLE
Newbie question about installation

### DISCUSSION THREAD

**Discussion by MarksenDE** [outline/outline#4252] [Self Hosting]
Hi all,
i'm absolute new at Outline and want to give it a try. I'm working on a Ubuntu 20.04. Server and have installed all requirements. After copied the whole Outline-framework to the server i've started with "yarn install --frozen-lockfile && yarn build" and get the error:

"yarn: error: no such option: --frozen-lockfile"

Any idea what i'm doing wrong?

Thanks and br

Mark

**Comment by Limezy** [comment]
Hi ! Completing a fully working setup of outline including storage, SSL and authentication can be quite hard. This is why I would rather recommend to go for a "one click" install option or very detailed tutorial. Several have been provided by the community already (you can search in the discussions). Personally I have packaged Outline as a Yunohost package 📦 https://github.com/YunoHost-Apps/outline_ynh so you can just setup quickly a Yunohost server and install it that way !

**Reply by MarksenDE** [reply]
Thanks Limezy,

unfortunately i can follow your link but can't create a package because a personal gmail address isn't allowed, so it's really complicated to become a selfhosted version working - i'll try next year again.

- request closed -

**Comment by almereyda** [comment]
Hi @MarksenDE, instead of building from source, have you tried to deploy Outline with Docker Compose? It is much more deterministic than supporting the many degrees of freedom that a custom operating system brings.

There are quite a few tutorials linked from the discussions here, plus an exemplary setup within the official documentation:

- https://app.getoutline.com/s/770a97da-13e5-401e-9f8a-37949c19f97e/doc/docker-7pfeLP5a8t

> Docker is the recommended way to run Outline
